<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="4" rowspan="9" colspan="2">
       <widget class="QCheckBox" name="connectionChckB">
        <property name="text">
         <string>Connected</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="11">
       <widget class="QwtKnob" name="pitchSetAngleKnb">
        <property name="lowerBound">
         <double>-90.000000000000000</double>
        </property>
        <property name="upperBound">
         <double>90.000000000000000</double>
        </property>
        <property name="scaleMaxMajor">
         <number>5</number>
        </property>
        <property name="totalSteps">
         <UInt>1000</UInt>
        </property>
        <property name="knobStyle">
         <enum>QwtKnob::Raised</enum>
        </property>
        <property name="totalAngle">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="4" colspan="2">
       <widget class="QLabel" name="serialLbl">
        <property name="text">
         <string>Enter serial params:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="4" colspan="2">
       <widget class="QPushButton" name="connectionBtn">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="rollLbl">
        <property name="text">
         <string>Roll</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="pitchLbl">
        <property name="text">
         <string>Pitch</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="4">
       <widget class="QComboBox" name="serialSpeedBox">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" rowspan="11">
       <widget class="QwtKnob" name="rollSetAngleKnb">
        <property name="lowerBound">
         <double>-90.000000000000000</double>
        </property>
        <property name="upperBound">
         <double>90.000000000000000</double>
        </property>
        <property name="totalSteps">
         <UInt>1000</UInt>
        </property>
        <property name="totalAngle">
         <double>180.000000000000000</double>
        </property>
        <property name="borderWidth">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="pitchSetAngleDgtl">
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="13" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="rollSetAngleDgtl">
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="4">
       <widget class="QLineEdit" name="serialNameFld">
        <property name="text">
         <string>/dev/pts/12</string>
        </property>
        <property name="maxLength">
         <number>64</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pitchSetAngleDgtl</sender>
   <signal>valueChanged(double)</signal>
   <receiver>pitchSetAngleKnb</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pitchSetAngleKnb</sender>
   <signal>valueChanged(double)</signal>
   <receiver>pitchSetAngleDgtl</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rollSetAngleDgtl</sender>
   <signal>valueChanged(double)</signal>
   <receiver>rollSetAngleKnb</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rollSetAngleKnb</sender>
   <signal>valueChanged(double)</signal>
   <receiver>rollSetAngleDgtl</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
